core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -165,6 +166,19 @@ angular.module('oppia').directive('storyNodeEditor', [
             return '/skill_editor/' + skillId;
           };
 
+          $scope.getPrerequisiteSkillsDescription = function() {
+            let skills = $scope.getPrerequisiteSkillIds();
```

Comment:
Prefer `const` if the value does not change.

Reply:
Done!

Reply:
Done!